crates/client/rpc/src/methods/read/get_block_with_tx_hashes.rs

```diff
@@ -63,3 +63,86 @@ pub fn get_block_with_tx_hashes(
         }
     }
 }
+
+#[cfg(test)]
+mod tests {
+    use super::*;
+    use crate::test_utils::{make_sample_chain_1, open_testing, SampleChain1};
+    use rstest::rstest;
+    use starknet_core::types::{BlockTag, Felt, L1DataAvailabilityMode, ResourcePrice};
+
+    #[rstest]
+    fn test_get_block_with_tx_hashes() {
+        let _ = env_logger::builder().is_test(true).try_init();
+        let (backend, rpc) = open_testing();
+        let SampleChain1 { block_hashes, tx_hashes, .. } = make_sample_chain_1(&backend);
```

Comment:
same as above. currently it looks like we're checking db storage and not the RPC logic itself or maybe I am missing something